hc2/external/elfio/elfio_amdgpu.hpp

```diff
@@ -0,0 +1,89 @@
+//===----------------------------------------------------------------------===//
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef ELFIO_AMDGPU_HPP
+#define ELFIO_AMDGPU_HPP
+
+namespace hc {
+
+// AMDGPU specific e_flags.
```

Comment:
could you add a comment to document where these hex values are defined/coming from?

Reply:
Good point, just added it.